.changeset/introduce_includes.md

```diff
@@ -1,5 +1,5 @@
 ---
-"@biomejs/biome": minor
+"@biomejs/biome": major
```

Comment:
This is a `minor`. Instead you should create a new changeset that mentions that `ignore` was removed (and list all the configuration fields removed) and `includes` should be used instead. Ideally, we should tell them to use `biome migrate`, and even tell them how an `ignore` value is translated in the new `includes` field.

Reply:
The changes are linked. Why should we have two distinct changesets? If we have two, how can I create a hyperlink to the other?

Reply:
I see two options:
- a changeset that talks about of the **new** `includes` (minor), and one that talks about the **removed** `ignore` (major).
- a changeset (major) that talks about the removal of `include` and `ignore` in favour of `includes`. 

It seems that you're more keen to use the second option. If so, this changeset should be updated to highlight the fact that `ignore` (and later `include`) are removed, how they can be replaced, etc.

You can't link two changesets. Maybe we can link a possible migration guide for the website.